### PR TITLE
calculate

### PR DESCRIPTION
**Pull request by kasiasieradzka**

### CODE REVIEW

File: module1/homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -2,6 +2,22 @@
...
+       return std::to_string(first - second);
+       }
+    else if (command == "multiply"){
+       return std::to_string(first * second);
+       }
+     else if (command == "divide"){
+       if (second == 0){
+          return "Division by 0";
+          }
+       else
```
Comment (DamiXioR) [inline module1/homework/calculate/calculate.hpp:18]: last else is unnecessary

Comment (nauka-programowania-MA) [inline module1/homework/calculate/calculate.hpp:18]: use space instead tab. Or change in your IDE to put spaces, when you use tab

Comment (kasiasieradzka) [reply module1/homework/calculate/calculate.hpp:18]: thanks

Comment (kasiasieradzka) [reply module1/homework/calculate/calculate.hpp]: done

**Comment by ziobron** [comment]
{ "requirement_ids": [1119, 1429], "points": [2, 5] }

**Pull request closed without merging**